src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/ConnectHelper.cs

```diff
@@ -4,7 +4,9 @@
 
 using System.Diagnostics;
 using System.IO;
+#if !BROWSER_DOES_NOT_SUPPORT_QUIC
```

Comment:
Yes definitely right about the double negative.

Reply:
Why is this file even included in the build, since SocketsHttpHandler isn't supported when targeting the browser and a dummy stub file is used instead?

Reply:
This one and a couple of others are being used in `HttpClient.cs`.

Reply:
Removed the dependency of the `ConnectHelper.cs` and `CancellationHelper.cs` in the code from Browser.  See recent commits.